.storybook/guides/develop.mdx

```diff
@@ -437,18 +437,23 @@ Default.parameters = {
 };
 ```
 
-If a story should not be shown in the sidebar because it is only for:
+#### Story tags
 
-- Documentation purposes, you can use the tags:
+Through this project's global configuration, all stories include the `autodocs` tag by default, which causes them to appear on the component's "Docs" page. The `dev` tag, which Storybook applies by default, denotes stories that should be included in the Storybook sidebar.
+A story's [tags](https://storybook.js.org/docs/writing-stories/tags#built-in-tags) can be configured for a few different purposes. The `!` symbol is used to negate the tag:
+
+- If a story should not be shown in the sidebar:
   ```js
-  Default.tags = ["autodocs", "!dev"];
+  Default.tags = ["!dev"];
   ```
-- Visual regression testing, you can use the following tags:
+- If a story should not be shown on the component's "Docs" page:
   ```js
-  Default.tags = ["test", "!autodocs", "!dev"];
```

Comment:
I removed the mentions of the `test` tag here, as the Storybook docs say the test tag is for Storybook's test runner:
>Stories tagged with test do not currently affect Storybook's UI, but can be used to filter the [test runner](https://storybook.js.org/docs/writing-tests/test-runner#run-tests-for-a-subset-of-stories).

The `test` tag is also already applied by default to all stories in Storybook.

Reply:
Do we want to include removing the "test" tag in this PR from stories where they are using them now?

Reply:
Sure. I've removed them in an added commit, [7df05a1](https://github.com/adobe/spectrum-css/pull/2924/commits/7df05a19867ba66df2ae7751636b0946543acb79)